client/i18n/locales/english/intro.json

```diff
@@ -6137,13 +7080,105 @@
+        ]
+      },
+      "node-package-manager": {
+        "note": "Coming Late 2026",
+        "intro": [
+          "In this module, you will be introduced to the Node Package Manager, which developers use to manage project dependencies and scripts. Then you will get to practice what you have learned through workshops and labs and test your knowledge with a short quiz."
+        ]
+      },
+      "http-and-the-web-standards-model": {
+        "note": "Coming Late 2026",
+        "intro": [
+          "In this module, you will learn about HTTP (Hypertext Transfer Protocol) and other important concepts including the request-response model, common response codes, DNS, TCP/IP, and more. Then you will get to practice what you have learned through a build your own web server workshop and test your knowledge with a short quiz."
+        ]
+      },
+      "rest-api-and-web-services": {
+        "note": "Coming Late 2026",
+        "intro": [
+          "In this module, you will learn about the REST API (Representational State Transfer Application Programming Interface) and how microservices work. Then you will take a short quiz to test your knowledge."
+        ]
+      },
+      "introduction-to-express": {
+        "note": "Coming Late 2026",
+        "intro": [
+          "In this module, you will be introduced to Express.js, which is a framework used to build RESTful APIs. Then you will practice your skills through workshops and labs and test your knowledge with a short quiz."
+        ]
+      },
+      "express-middleware": {
+        "note": "Coming Late 2026",
+        "intro": [
+          "In this module, you will learn about middleware in Express.js, which is used to handle requests and responses between the client and server. You will then practice your skills through a workshop and lab and test your knowledge with a short quiz."
+        ]
+      },
+      "error-handling-in-express": {
+        "note": "Coming Late 2026",
+        "intro": [
+          "In this module, you will learn about how error handling, debugging, and health checks work in Express.js. You will then practice what you have learned in a lab and test your knowledge with a short quiz."
+        ]
+      },
+      "websockets": {
+        "note": "Coming Late 2026",
+        "intro": [
+          "In this module, you will be introduced to websockets, which is a protocol used for real time communication with the client and server. You will then practice what you have learned in labs and workshops and test your knowledge with a short quiz."
+        ]
+      },
+      "node-and-sql": {
+        "note": "Coming Late 2026",
+        "intro": [
+          "In this module, you will practice building applications with Node and SQL. Then you will take a short quiz to test your knowledge."
+        ]
+      },
+      "security-and-privacy": {
+        "note": "Coming Fall 2025",
```

Comment:
This was copied from FSD, but I noticed the timeline doesn't match the other blocks. I'm wondering if it should be changed to Late 2026 instead.

Reply:
yeah, all backend js should be updated to late 2026